packages/zowe-explorer-api/src/logger/IZoweLogger.ts

```diff
@@ -76,25 +77,25 @@ export class IZoweLogger {
      * Log an error to the Imperative log
      *
      */
-    public logImperativeMessage(message: string, severity: MessageSeverityEnum): void {
```

Comment:
wondering if we should
```typescript
public logImperativeMessage(message: string, severity: MessageSeverityEnum);
public logImperativeMessage(message: string, severity: MessageSeverity) { ... }
```

Reply:
Thinking that this will be covered by TypeScript as the deprecated type is now aliasing the renamed enum, but can do some local testing to confirm.